snowshu/core/configuration_parser.py

```diff
@@ -94,8 +98,41 @@ def _get_dict_from_anything(self, dict_like_object:Union[str,'StringIO',dict])->
             try:
                 return yaml.safe_load(dict_like_object.read())
             except AttributeError:
-                with open(dict_like_object,'r') as f:
-                    return yaml.safe_load(f.read())
+                return self._load_and_verify_file(dict_like_object, **kwargs)
```

Comment:
Is it possible to verify any input here against the jsonschema instead of just the actual file case? Essentially, generate the `instance` object here, and then pass it into the verifying function.

Not sure if that situation is actually possible though.